docs/user/benchmarking.md

```diff
@@ -0,0 +1,89 @@
+# Benchmarking
+
+This guide explains how to write and run garage benchmark scripts.
+
+## Prerequisites
+
+Firstly, you should have installed garage. If not, you can see instructions in the [latest documentation](https://garage.readthedocs.io/en/latest/user/installation.html) to install garage.
+
+Install `google-cloud-storage`:
+
+`pip install google-cloud-storage`
+
+## Install `garage-benchmarks`
+
+`cd garage/benchmarks && pip install -e .`
+
+## Write garage benchmark scripts
+
+**Define an experiment function**
+
+Use `@wrap_experiment` to define your experiment. `ctxt` is used for `@wrap_experiment`.
+The second and third parameters of the function should be `env_id` and `seed`.
+You should give your function a good name because it is used as the label name when it comes to plotting.
+See more [examples](https://github.com/rlworkgroup/garage/tree/master/benchmarks/src/garage_benchmarks/experiments).
+```py
+@wrap_experiment
+def trpo_garage_pytorch(ctxt, env_id, seed):
+    ...
+```
+
+**Define an execution function using `@benchmark` and `iterate_experiments()`**
+
+Plotting is enabled by default. You can explicitly disable it by defining `@benchmark(plot=False)`. Automatic benchmarking is diabled by default. You can explicitly disable it by defining `@benchmark(auto=True)`.
```

Comment:
```suggestion
Plotting is enabled by default. You can explicitly disable it by defining `@benchmark(plot=False)`. Automatic benchmarking is disabled by default. You can explicitly enable it by defining `@benchmark(auto=True)`.
```

Also, I guess this demands explaining what Automatic benchmarking is and why someone would want to enable it. Since these are user docs, maybe we don't need to mention automatic benchmarking here at all, but I am not sure about that opinion.

Reply:
Thanks for pointing out this. I will remove Automatic benchmarking.